readme.txt

```diff
@@ -1,3 +1,18 @@
 This repository contains a Meson build definition for project mt32emu.
 
 For more information please see http://mesonbuild.com.
+
+For more information about mt32emu library see https://github.com/munt/munt.
+
+To use this wrap:
+
+    $ meson wrap install mt32emu
+    $ echo 'munt-libmt32emu*' >> subprojects/.gitignore
```

Comment:
The better way to do this, JFYI, is to have this in your .gitignore:
```txt
# Ignore all files and directories
*
# But allow .wrap files to be tracked
!*.wrap
# Also allow .gitignore files to be tracked.
!.gitignore
```

Reply:
That's certainly handy… but I don't know what other users might be storing in their `subprojects/` dir. Documentation for their wraps perhaps? `.gitattributes`? Other files or directories with sources inside? Personally, I have  `packagecache` dir in .gitignore, but I'm starting to think that was a mistake.

My intention in here was to communicate: "hey, Meson is going to place a file like this in subprojects/ in case you haven't used this feature before" and I trust people to adapt it to their projects :)

Reply:
Sure, for the common case of "I have a bunch of wrap files" this is a pretty hand trick. Maybe we should put this in the FAQ or "how do i?" documentation